### DISCUSSION TITLE
Horrible Choices and Somehow Worse Choice

### DISCUSSION THREAD

**Discussion by ItWorkedLastTime** [Snark-Industries/CenterStage#3] [Polls]
bad choices

**Comment by OrioMann** [comment]
Raw whale Fr Fr, I would make sushi out of dem whales

**Reply by ItWorkedLastTime** [reply]
No it cannot be eaten as anything other than an 8 oz chunk of whale that was just cut off a whale

**Reply by OrioMann** [reply]
Did I studder?

**Comment by ItWorkedLastTime** [comment]
![image](https://github.com/Snark-Industries/CenterStage/assets/145297378/80932c49-d210-4ed1-83fc-5d9cf1e6a1be)
